src/main/java/org/broadinstitute/hellbender/tools/walkers/sv/JointGermlineCNVSegmentation.java

```diff
@@ -682,14 +680,18 @@ private static Genotype prepareGenotype(final Genotype g, final Allele refAllele
 
     private static void correctGenotypePloidy(final GenotypeBuilder builder, final Genotype g, final int ploidy,
```

Comment:
I know I dropped the ball, but can you add some comments here?  This is for overlapping events, right?

Reply:
Done. This just modifies the genotypes for input variants so the ploidies are consistent with the ped file and also irons out any no-call/null GTs.